### ISSUE TITLE
Player names are visible when custom is not full

### ISSUE THREAD

**Issue by thekeenant** [thekeenant/Tabbed#10]
![image](https://cloud.githubusercontent.com/assets/1189775/14763816/9e709292-0967-11e6-9ede-49b0d21a9f0d.png)

``` java
TableTabList tab = (TableTabList) Atlas.getTabbed().getTabList(player);
if (tab == null)
  tab = Atlas.getTabbed().newTableTabList(player, 3);
tab.set(0, 1, new TextTabItem("Boop"));
```

They only appear when the tab list isn't 4x20.

**Comment by thekeenant** [comment]
Somehow Tabbed will have to cancel vanilla `PlayerInfoData` sent, though not cancelling them prior to a player is spawned.

**Comment by thekeenant** [comment]
Possible solution?
1. Listen to player info data sent
2. If it starts with `$` or is not `ADD_PLAYER` then ignore
3. Schedule `REMOVE_PLAYER` a few ticks later

And also every second or so send every player a vanilla player info packet `ADD_PLAYER` for every player online.

**Comment by thekeenant** [comment]
Resolved since 1.8.
